target/linux/mvebu/base-files/etc/board.d/02_network

```diff
@@ -28,6 +28,9 @@ armada-385-turris-omnia)
 	ucidef_set_interface_lan "lan0 lan1 lan2 lan3 lan4"
 	ucidef_set_interface_wan "eth2"
 	;;
+armada-385-wd-mcex2u)
+	ucidef_set_interface_lan "eth0" "dhcp"
```

Comment:
I'm in favour of this as it "makes sense" but looking https://github.com/openwrt/openwrt/pull/1062 it's still a mess....

Reply:
Well, I checked the thread. While, in general, I tend to agree with your approach to make one-port devices DHCP-driven by default, I could see at least one somewhat valid point against: OpenWrt does require configuring after installation and looking for the device could be problematic. But on the other hand I could see static approach to be even more problematic in case there is already an OpenWrt router serving such one-port device like NAS (easily imaginable situation, actually). 

Still, I think that, regardless of overall stance on "dhcp by default for one-ports" is being undefined, using it on a particular device, with a known upgrade workflow, should be acceptable.

Like it is in this case — upgrade procedure could be done via SSH on the original firmware, user is aware of the IP, MAC address is properly inherited by the OpenWrt firmware after upgrade. So, considering the principle of least surprise, I'd prefer to leave as it is.